scripts/finetune.py

```diff
@@ -230,6 +230,9 @@ def train(
     )
     # TODO do we need this fix? https://huggingface.co/docs/accelerate/usage_guides/fsdp#saving-and-loading
     trainer.save_model(cfg.output_dir)
```

Comment:
should we if/else with `trainer.save_model(cfg.output_dir)` ? I expect saving the model if using lora is a pointless exercise since that would simply be the base model?

Reply:
Ah, I see! I thought `save_model` would at least output the checkpoint which could be used to train further.

Reply:
Would this be ok?

Reply:
so https://github.com/tloen/alpaca-lora/blob/main/finetune.py#L275 and https://github.com/johnsmith0031/alpaca_lora_4bit/blob/main/finetune.py#L187 both use save_pretained, but I forget why I changed it to `save_model`. thoughts?

Reply:
Feel free to edit the PR :) It's open to edits.